doc/pages/mapping.asciidoc

```diff
@@ -159,19 +159,37 @@ In normal mode:
 Shift version of those mappings exist as well
 (for example `<s-left>` maps to `H`).
 
+
 == Additional notes
 
-Some keys cannot be mapped directly but they can still mapped by `c-v` to directly insert their keycode as an argument to map.
+Some keys cannot be mapped directly but they can still mapped by either: typing `c-v` before the key to insert the keycode directly or by using an alias.
```

Comment:
You cannot map these keys via `<c-v>`, you have to use their actual names.

Reply:
what do you mean "You cannot map these keys via `<c-v>`..."? 

I have mapped `<c-/>` exactly as described here https://github.com/mawww/kakoune/issues/2553#issuecomment-437600134 i.e. I used `<c-v><c-/>` to input a symbol in to `map global normal...`

this worked for me in Kakoune v2020.09.01

Reply:
In the current Git version of Kakoune, you can't map these keys with `<c-v>`, it's changed since v2020.09.01.